docs/usb/wdi-usb-interface.md

```diff
@@ -33,13 +38,29 @@ https://gitlab.freedesktop.org/libevdev/libevdev/-/blob/master/include/linux/lin
  BTN_THUMBR
 ```
 
-## Drive actions
+## Drive Actions
 | Action | Keyboard Controls | Gamepad Controls |
 |--------|------|-----|
 | Drive forward | 'w' \| up arrow | Y axis (towards min val) |
 | Drive backward | 's' \| down arrow | Y axis (towards max val) |
 | Drive left | 'a' \| left arrow | X axis (towards min val) |
 | Drive right | 'd' \| right arrow | X axis (towards max val) |
-| Emergency stop | backspace | button_east?? would like this to be something that is easy to press under stress |
-| Enable drive | enter | start button |
-| 
\ No newline at end of file
+| Emergency stop | backspace | BTN_SOUTH |
+| Enable device control | enter | BTN_START |
+| Override | caps lock | BTN_EAST |
```

Comment:
Per discussion with @jeredhdean, some buttons will be left open so that different implementations of the WDI can do map those buttons depending on what it needs to do. Override would be a LUCI-specific mapping of this button.